patientsearch/models/sync.py

```diff
@@ -53,19 +53,23 @@ def HAPI_request(
         # meaning new patients won't immediately appear in results.
         # Disable caching until we find the need and safe use cases
         headers = {'Cache-Control': 'no-cache'}
-        resp = requests.get(
-            url, auth=BearerAuth(token), headers=headers, params=params)
+        try:
+            resp = requests.get(
+                url, auth=BearerAuth(token), headers=headers, params=params)
+        except requests.exceptions.ConnectionError as error:
+            current_app.logger.exception(error)
+            raise RuntimeError("EMR FHIR store inaccessible")
     elif VERB == 'POST':
         resp = requests.post(url, auth=BearerAuth(token), json=resource)
     elif VERB == 'PUT':
         resp = requests.put(url, auth=BearerAuth(token), json=resource)
     elif VERB == 'DELETE':
```

Comment:
out of scope for this PR, but [I discovered](https://github.com/uwcirg/jwt-proxy/blob/main/jwt_proxy/api.py#L12-L13) you can use `requests.request(method=VERB...)` instead of needing to call each different `requests` function

Reply:
oh nice!